scripts/fireupgrade.sh

```diff
@@ -166,7 +166,6 @@ done
 if [[ $rc -ne 0 ]]
 then
     /home/pi/firewalla/scripts/firelog -t local -m "FIREWALLA.UPGRADE($mode) Starting RECOVER NETWORK "+`date`
-    external_script='sudo  CHECK_FIX_NETWORK_REBOOT=no CHECK_FIX_NETWORK_RETRY=no /home/pi/firewalla/scripts/check_fix_network.sh'
```

Comment:
why delete this?

Reply:
It's redundant if you check the following lines
https://github.com/firewalla/firewalla/blob/94b99c4697a7bb9526bd3ac172d2fffcf59584bf/scripts/fireupgrade.sh#L169-L175